course_catalog/constants.py

```diff
@@ -116,11 +116,13 @@ class PrivacyLevel(Enum):
     "Computer Science": ["Computer Science"],
     "Crisis Management": ["Management"],
     "Data Modeling & Analytics": ["Computer Science"],
+    "Digital Transformation": ["Business", "Information Technology"],
     "Design & Manufacturing": ["Systems Engineering"],
     "Energy & Sustainability": ["Energy", "Earth Science"],
     "Imaging": ["Computer Science", "Electrical Engineering"],
     "Innovation": ["Innovation"],
     "Leadership & Communication": ["Leadership", "Communication"],
+    "Legal Tech": ["Social Science", "Legal Studies"],
```

Comment:
```suggestion
    "Legal Tech": ["Information Technology", "Legal Studies"],
```

Based on the courses, I don't think these are a broad match for "Social Science" either. But I do think ""nformation Technology" is a fair match.